app/backend/wells/management/commands/legacy_records.py

```diff
@@ -0,0 +1,76 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import logging
+from timeit import default_timer as timer
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from wells.models import Well
+from wells.stack import StackWells
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Run from command line:
+    python manage.py legacy_records
+    """
+
+    def add_arguments(self, parser):
+        # Arguments added for debugging purposes.
+        # e.g. don't cleanup, don't upload: python manage.py export_databc --cleanup=0 --upload=0
+        parser.add_argument('--start', type=int, nargs='?', help='Well to start at', default=1)
+        parser.add_argument('--end', type=int, nargs='?', help='Well to end at', default=50)
```

Comment:
This looks good to me, but just thinking about running this command in batches, it might help to have an option to just update the next `n` wells ordered by well tag.

Reply:
The start and end range is easier to compute to a list of well_tag_numbers without an activity submission. If there are `--start` and `--amount` flags then wouldn't the `--amount` imply the number of "well's without an activity submission"? I don't want someone to assume that --amount=5 will actually process 5 wells when in reality the next 5 wells only has 2 without an activity submission.

Reply:
True, but I'm just thinking in terms of running a series of jobs "process the next 1000 wells that need submission history" 100 times vs "process well 7001 to 8000" then "process 8001 to 9000", then "process 9001 to 10000" etc.

Also, --amount=5 should still process 5 wells (unless we run out of wells to process), but possibly well tag numbers 1 through 10 not 1 through 5.

Reply:
I've added a `--next=` argument which (optionally) uses `--start` to process X number of records.